js/web/docs/webgpu-operators.md

```diff
@@ -61,7 +62,7 @@ Do not modify directly.*
 | ReduceSumSquare | ai.onnx(1-10,11-12,13-17,18+) |  |
 | Relu | ai.onnx(6-12,13,14+) |  |
 | Reshape | ai.onnx(5-12,13,14+) | no GPU kernel |
-| Resize | ai.onnx(10,11-12,13-17,18,19+); com.ms.internal.nhwc(10,11-12,13-17,18,19+) | CoordinateTransformMode align_corners is not supported with downsampling |
+| Resize | ai.onnx(10,11-12,13-17,18,19+); com.ms.internal.nhwc(10,11-12,13-17,18,19+) |  |
```

Comment:
Same here

Reply:
I did not update it manually, just used `npm run build:doc` as otherwise the pipeline does not pass

Reply:
<img width="647" alt="image" src="https://github.com/microsoft/onnxruntime/assets/7466130/03c5c68e-5511-41e5-a3a1-306282d26762">

Reply:
did you run `npm run prepare` command in onnxruntime\js\web directory?

Reply:
Please try running `npm run prepare` in onnxruntime\js\web folder also. It seems you ran it in onnxruntime\js I am not sure it is supposed to run in the subdirectories.

Reply:
Does not help. It started to remove comments recently, after I've merged main branch. Before recent updates it worked as expected
<img width="891" alt="image" src="https://github.com/microsoft/onnxruntime/assets/7466130/a23dc1c3-9a39-4c2d-8820-ee76afae1039">

Reply:
I have seen this error a while ago. I think running `npm ci` under onnxruntime\js, onnxruntime\js\common, and onnxruntime\js\web directories as mentioned [here](https://gist.github.com/fs-eire/a55b2c7e10a6864b9602c279b8b75dce) might help

Reply:
Yeah, that helped, thanks